utils.sh

```diff
@@ -39,7 +39,7 @@ function create_cluster() {
     export TF_LOG=DEBUG
 
     cp ${assets_dir}/install-config.yaml{,.tmp}
-    "${assets_dir}/openshift-install" --dir "${assets_dir}" --log-level=debug create manifests
+    $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug create manifests
```

Comment:
Hm, slight change in behavior here.  The change assumes ${assets_dir} is `ocp`.  That's hard coded elsewhere right now, so I guess it's fine.